synapse/rest/admin/background_updates.py

```diff
@@ -29,37 +34,38 @@
 class BackgroundUpdateEnabledRestServlet(RestServlet):
     """Allows temporarily disabling background updates"""
 
-    PATTERNS = admin_patterns("/background_updates/enabled")
+    PATTERNS = admin_patterns("/background_updates/enabled$")
 
     def __init__(self, hs: "HomeServer"):
-        self.group_server = hs.get_groups_server_handler()
-        self.is_mine_id = hs.is_mine_id
-        self.auth = hs.get_auth()
+        self._is_mine_id = hs.is_mine_id
```

Comment:
None of these updates seem to `_is_mind_id`, I think we can remove it everywhere (by the way, thank you for cleaning up some of this code while you were in here!)